input/templates/my-account-address-book.hbs

```diff
@@ -34,61 +34,34 @@
                       <div class="box-addresses-title">
                         {{i18n "my-account:addressBook.defaultAddresses.title"}}
                       </div>
+                      {{#if content.defaultShippingAddress}}
```

Comment:
It is possible that the address is not set, in which case we have to make sure the box is not appearing. Actually, it would even be good that some message appears instead, but well, something for the future :P